Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/qrcode/OrganizerWarnQrCodeScannerFragment.kt

```diff
@@ -31,7 +30,8 @@ class OrganizerWarnQrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_sca
     @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
     private val viewModel: OrganizerWarnQrCodeScannerViewModel by cwaViewModels { viewModelFactory }
 
-    private val binding: FragmentQrcodeScannerBinding by viewBinding()
```

Comment:
We use this delegate for the binding everywhere, why doesn't this work here?

Reply:
Honestly I'm not sure why. All I know is we are using the same layout for the universal scanner and for this `OrganizerWarnOthers` scanner. The bug is related to the binding of the layout and it works with the universal scanner but doesn't work in the organizer one. The universal scanner had this binding initialization change #4788 so I'm trying to see if this will fix the bug once an RC is built.

Reply:
Please add a comment that this type of binding initialisation doesn't get removed accidently